CPAC/distortion_correction/distortion_correction.py

```diff
@@ -498,19 +548,18 @@ def connect_distortion_correction(workflow, strat_list, c, diff, blip,
             workflow.connect(match_epi_fmaps_node, 'same_pe_epi',
                              blip_correct, 'inputspec.same_pe_epi')
 
-        if "None" in c.distortion_correction:
+        if False in c.functional_preproc['distortion_correction']['run']:
```

Comment:
So the only [concern I have left](https://github.com/FCP-INDI/C-PAC/pull/1400#issue-519385702) before merging this PR into [`enh/nested_config`](https://github.com/FCP-INDI/C-PAC/tree/enh/nested_config) is 

> * This block seems one level not-indented-far-enough: https://github.com/FCP-INDI/C-PAC/blob/5e3581e04c1357af797f46482d75eaa368d3f84c/CPAC/distortion_correction/distortion_correction.py#L551-L563 If I'm reading the code correctly, `strat` is defined in this loop https://github.com/FCP-INDI/C-PAC/blob/5e3581e04c1357af797f46482d75eaa368d3f84c/CPAC/distortion_correction/distortion_correction.py#L494 so the whole block in question here would only apply to the final iteration of that loop. Possibly related: #1072, #1131, #1242

Reply:
Resolved in #1423